lib/core/auth.js

```diff
@@ -369,7 +370,7 @@ export default class Auth {
 
     if (process.client) {
       if (noRouter) {
-        window.location.replace(to)
+        window.location.replace(this._baseUrl + to)
```

Comment:
What if _baseUrl ends in a slash? isn't that going to cause an invalid url?

Reply:
Thank!! Good catch. This issue can be resolve in the [constructor](https://github.com/nuxt-community/auth-module/pull/523/files#diff-9a2f0794afc86382d1c3c12125bd13f4R8)